### ISSUE TITLE
Toolbar buttons no longer display

### ISSUE THREAD

**Issue by 3rror404** [tjvantoll/nativescript-IQKeyboardManager#24]
Hi @tjvantoll,

since updating to ~~N3.x~~ iOS 11 the toolbar buttons (done/prev/next) no longer display. Tapping the area where they should be triggers the correct action but they are completely invisible.

I see a lot of warnings relating to IQKeyboardManager during build:

```
=== BUILD TARGET CocoaLumberjack OF PROJECT Pods WITH CONFIGURATION Debug ===
In file included from ++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardManager.m:26:
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.h:180:9: warning: 
      parameter 'shouldShowPlaceholder' not found in the function declaration [-Wdocumentation]
 @param shouldShowPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
        ^~~~~~~~~~~~~~~~~~~~~
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.h:180:9: note: 
      did you mean 'showPlaceholder'?
 @param shouldShowPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
        ^~~~~~~~~~~~~~~~~~~~~
        showPlaceholder
1 warning generated.
In file included from ++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardReturnKeyHandler.m:24:
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardReturnKeyHandler.h:88:9: warning: 
      parameter 'UIView' not found in the function declaration [-Wdocumentation]
 @param UIView object to register all it's responder subviews.
        ^~~~~~
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardReturnKeyHandler.h:88:9: note: 
      did you mean 'view'?
 @param UIView object to register all it's responder subviews.
        ^~~~~~
        view
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardReturnKeyHandler.h:95:9: warning: 
      parameter 'UIView' not found in the function declaration [-Wdocumentation]
 @param UIView object to unregister all it's responder subviews.
        ^~~~~~
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardReturnKeyHandler.h:95:9: note: 
      did you mean 'view'?
 @param UIView object to unregister all it's responder subviews.
        ^~~~~~
        view
2 warnings generated.
=== BUILD TARGET CocoaLumberjack OF PROJECT Pods WITH CONFIGURATION Debug ===
In file included from ++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.m:25:
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.h:180:9: warning: 
      parameter 'shouldShowPlaceholder' not found in the function declaration [-Wdocumentation]
 @param shouldShowPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
        ^~~~~~~~~~~~~~~~~~~~~
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.h:180:9: note: 
      did you mean 'showPlaceholder'?
 @param shouldShowPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
        ^~~~~~~~~~~~~~~~~~~~~
        showPlaceholder
1 warning generated.
=== BUILD TARGET CocoaLumberjack OF PROJECT Pods WITH CONFIGURATION Debug ===
In file included from ++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardManager.m:26:
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.h:180:9: warning: 
      parameter 'shouldShowPlaceholder' not found in the function declaration [-Wdocumentation]
 @param shouldShowPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
        ^~~~~~~~~~~~~~~~~~~~~
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.h:180:9: note: 
      did you mean 'showPlaceholder'?
 @param shouldShowPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
        ^~~~~~~~~~~~~~~~~~~~~
        showPlaceholder
1 warning generated.
In file included from ++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardReturnKeyHandler.m:24:
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardReturnKeyHandler.h:88:9: warning: 
      parameter 'UIView' not found in the function declaration [-Wdocumentation]
 @param UIView object to register all it's responder subviews.
        ^~~~~~
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardReturnKeyHandler.h:88:9: note: 
      did you mean 'view'?
 @param UIView object to register all it's responder subviews.
        ^~~~~~
        view
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardReturnKeyHandler.h:95:9: warning: 
      parameter 'UIView' not found in the function declaration [-Wdocumentation]
 @param UIView object to unregister all it's responder subviews.
        ^~~~~~
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQKeyboardReturnKeyHandler.h:95:9: note: 
      did you mean 'view'?
 @param UIView object to unregister all it's responder subviews.
        ^~~~~~
        view
2 warnings generated.
=== BUILD TARGET CocoaLumberjack OF PROJECT Pods WITH CONFIGURATION Debug ===
In file included from ++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.m:25:
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.h:180:9: warning: 
      parameter 'shouldShowPlaceholder' not found in the function declaration [-Wdocumentation]
 @param shouldShowPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
        ^~~~~~~~~~~~~~~~~~~~~
++PATH_TO_APP++/platforms/ios/Pods/IQKeyboardManager/IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.h:180:9: note: 
      did you mean 'showPlaceholder'?
 @param shouldShowPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
        ^~~~~~~~~~~~~~~~~~~~~
        showPlaceholder
1 warning generated.

```

Any ideas?

```
┌──────────────────┬─────────────────┬────────────────┬─────────────┐
│ Component        │ Current version │ Latest version │ Information │
│ nativescript     │ 3.2.1           │ 3.2.1          │ Up to date  │
│ tns-core-modules │ 3.2.0           │ 3.2.0          │ Up to date  │
│ tns-android      │ 3.2.0           │ 3.2.0          │ Up to date  │
│ tns-ios          │ 3.2.0           │ 3.2.0          │ Up to date  │
└──────────────────┴─────────────────┴────────────────┴─────────────┘
```
I found a few issues on the IQKeyboardManager repo (https://github.com/hackiftekhar/IQKeyboardManager/issues/1026) that suggest this has been fixed in an update.

How would I go about updating the plugin to use the latest version of the pod? 

I tried editing the podfile to point to v5.0.3 which looks to be the latest version `pod 'IQKeyboardManager', '~> 5.0.3'`, and then generate the typings [as described in your docs](https://github.com/tjvantoll/nativescript-IQKeyboardManager#maintainers) but this gives me all sorts of errors.

**Comment by 3rror404** [comment]
I got the update working eventually and can confirm that version 5.0.3 of IQKeyboardManager fixes this issue.

**Comment by glen-stephens** [comment]
@3rror404 Experiencing the same issue.  Can you share the details on how you got it working with 5.0.3?

**Comment by glen-stephens** [comment]
I got it working on 5.0.7.  I have the new typings file.  Don't mind updating this one if you want it.

**Comment by tjvantoll** [comment]
Hey all,

Sorry for the late reply here. @glen-stephens @3rror404 were you able to get the plugin working without warnings? I tried bringing in this PR https://github.com/tjvantoll/nativescript-IQKeyboardManager/pull/22, which does get the plugin working, but I’m still getting TypeScript warnings. If you can get me a PR that updates the versions and fixes the typing problems I’ll merge and push it out asap.

**Comment by glen-stephens** [comment]
Hey @tjvantoll 

Sorry for the late response. I created 2 pull requests, one for index.d.ts and one for the podfile. It is working great for me. Thanks!

**Comment by tjvantoll** [comment]
Thanks so much @glen-stephens! I merged in your PRs and published a new 1.2.0 version of the plugin to npm.
